data/data_loader.py

```diff
@@ -1,25 +1,26 @@
 import numpy as np
 import pandas as pd
 from typing import Tuple, Optional
-from data.credit_card_default import credit_card_default_adapter as credit_card_default_da
-from data import data_preprocessor as dp
+from data.credit_card_default import credit_card_default_loader
+from data import recourse_adapter
 
 
-def load_credit_card_default_dataset(only_continuous: bool = True, data_dir: Optional[str] = None) -> Tuple[pd.DataFrame, dp.Preprocessor]:
-    """Returns the training data and a preprocessor for the Credit Card Default dataset.
+def load_credit_card_default_dataset(only_continuous: bool = True, data_dir: Optional[str] = None) -> Tuple[pd.DataFrame, recourse_adapter.RecourseAdapter]:
+    """Returns the training data and a recourse adapter for the Credit Card Default dataset.
 
     Args:
         only_continuous: Whether to drop or recategorize categorical variables, leaving only continuous features.
         data_dir: An override for the directory to load the data from (or download it to if unavailable).
 
     Returns:
-        The Credit Card Default dataset and a fitted preprocessor."""
+        The Credit Card Default dataset and a fitted recourse adapter."""
     if data_dir is not None:
-        return credit_card_default_da.load_data(only_continuous=only_continuous, data_dir=data_dir)
+        return credit_card_default_loader.load_data(only_continuous=only_continuous, data_dir=data_dir)
     else:
-        return credit_card_default_da.load_data(only_continuous=only_continuous)
+        return credit_card_default_loader.load_data(only_continuous=only_continuous)
 
 
+# TODO(@jakeval): This will be removed in a later PR
 def filter_from_poi(dataset, poi, immutable_features=None, feature_tolerances=None):
```

Comment:
Add at least a single-line docstring to this function, please.

Reply:
This function isn't used and is removed in PR #17.